### ISSUE TITLE
VSCodeVim hijacked keyboard shortcuts and issue persists after uninstall.

### ISSUE THREAD

**Issue by allthetime** [VSCodeVim/Vim#4949]
Hi, I installed VSCodeVim but promptly removed it after realizing that it breaks very basic and useful VSCode functionality. (command+d doesn't work, project search command+shift+f doesn't work)

I have uninstalled the plugin but these shortcuts are still not working. It seems that the install/uninstall has left something behind.

**Comment by allthetime** [comment]
You can see here that the "find in files" shortcut condition has been deleted

![](https://i.imgur.com/drCSzzz.png)

**Comment by mhchen** [comment]
Can you screenshot your keyboard shortcuts when you search for `cmd d`?

Example:
![image](https://user-images.githubusercontent.com/1325549/84985173-22763300-b0f1-11ea-8a17-21f70271bf1d.png)

Even the hijacking behavior should only be applicable when the vim extension is active:

![image](https://user-images.githubusercontent.com/1325549/84985333-68cb9200-b0f1-11ea-8217-d6b4789e0491.png)

It never unbinds the default, so I'm not sure how this happened to your setup.

**Comment by allthetime** [comment]
@mhchen so, it doesn't unbind the default, but there were some new shortcuts that were overriding the defaults. I already deleted the new ones so I have nothing to screen shot. I'm not 100% now if it was this plugin that did it (but its the only thing I've installed/uninstalled lately) 

Does this plugin create new keyboard shortcuts that don't get cleaned up?

**Comment by berknam** [comment]
It shouldn't. And those shortcuts should have still worked when you installed this extension. Maybe you had some other extension that conflicted with it.
The ctrl/command+shift+f shortcut is not even handled by this extension so if it didn't work I don't think it was because of this extension.

**Comment by mhchen** [comment]
Yes, the first screenshot in my above comment is from after uninstalling VSCodeVim. It removed all the plugin shortcuts. Given the nature of how extension keybindings work, it's extraordinarily difficult for me to believe that an extension can either delete any existing keybindings (they are additive from my understanding - see https://code.visualstudio.com/api/references/contribution-points#contributes.keybindings), or leave any kind of keybindings around after they are uninstalled.
